back/app/models/notifications/status_change_of_your_initiative.rb

```diff
@@ -69,8 +69,8 @@ class StatusChangeOfYourInitiative < Notification
 
     def self.make_notifications_on(activity)
       initiative = activity.item
-      cosponsors_ids = initiative.cosponsors_initiatives.where(status: 'accepted').pluck(:user_id)
-      recipient_ids = [initiative&.author_id] + cosponsors_ids
+      cosponsor_ids = initiative.cosponsors_initiatives.where(status: 'accepted').pluck(:user_id)
+      recipient_ids = [initiative&.author_id] + cosponsor_ids
```

Comment:
Unrelated change. We decided to pass the IDs in an attribute named `cosponsor_ids`, so I'm making the namining consistent (avoiding the plural `cosponsors_`)